reference-service/src/main/resources/config/application.yml

```diff
@@ -44,7 +44,7 @@ spring:
     open-in-view: false
     database-platform: org.hibernate.dialect.MySQL5InnoDBDialect
     database: MYSQL
-    show-sql: true
+    show-sql: false
```

Comment:
show-sql is turned off in TCS & Profile. And I think we probably don't want to have so much sql logs on Prod/Stage.